app/models/campaign.rb

```diff
@@ -17,8 +17,10 @@
 
 #= Campaign model
 class Campaign < ActiveRecord::Base
-  has_many :campaigns_courses, class_name: CampaignsCourses
+  has_many :campaigns_courses, class_name: CampaignsCourses, dependent: :destroy
+  has_many :campaigns_users, class_name: CampaignsUsers, dependent: :destroy
   has_many :courses, through: :campaigns_courses
+  has_many :users, -> { distinct }, through: :campaigns_users
```

Comment:
I think 'users' is a dangerous name for this. If I weren't looking at the definition, I would assume it scopes all the users in all the courses, rather than just the organizers of the campaign itself.

If it's not going to be used at this point, we should leave it out entirely. If it is going to be used, let's name it 'campaign_managers' or something else that makes it clear that it's not going through courses.